content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,546 @@
+---
+title: Monitoring and Access Policies for HTTP Egress Traffic
+description: Describes how to configure Istio for monitoring and access policies of HTTP egress traffic.
+publishdate: 2018-06-15
+subtitle:
+attribution: Vadim Eisenberg and Ronen Schaffer
+weight: 86
+keywords: [egress, access-control, monitoring]
+---
+
+While the main focus of Istio is managing traffic between the microservices inside a service mesh, Istio can also  manage ingress (from outside into the mesh) and egress (from the mesh outwards) traffic. Istio can uniformly enforce access policies and aggregate telemetry data for mesh-internal, ingress and egress traffic.
```

Comment:
While Istio's main focus is management traffic between microservices...

Reply:
_management traffic between microservices_

or

_management of traffic between microservices_

or 

_managing traffic between microservices_

?

Reply:
management of traffic between microservices